### DISCUSSION TITLE
Build on windows

### DISCUSSION THREAD

**Discussion by dsisnero** [itamae-kitchen/mitamae#142] [Ideas]
Can you make it so the package builds on windows?

**Comment by k0kubun** [comment]
Sure, I'm open to merging a contribution to support Windows. I don't have a Windows environment to develop/test the feature, so it'd be nice if somebody who would use the feature files a PR for it.

**Comment by dsisnero** [comment]
do you know of an equivalent mruby module to mruby-etc that does the same thing for windows?
